### ISSUE TITLE
Git version tags

### ISSUE THREAD

**Issue by stepanets** [streadway/amqp#437]
Can you tag version of  your code, please?

**Comment by danrs** [comment]
I second this motion. Go imports relying on a commit hash are a hassle. Even just a v0.0.1 tag on the top of `master` would be better than nothing

**Comment by Axu1g** [comment]
yes please everybody wants to fixate their dependencies versions.

**Comment by FedeBev** [comment]
Definitely yes, please!

**Comment by bombsimon** [comment]
Duplicate of #412
